### PR TITLE
Add lambda  for approve

### PR DESCRIPTION
**Pull request by alesapin**
Changelog category (leave one):

- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: tests/ci/approve_lambda/app.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    "annvsh",       # technical writer, Yandex
+    "atereh",       # DOCSUP
+    "azat",
+    "bharatnc",     # Newbie, but already with many contributions.
+    "bobrik",       # Seasoned contributor, CloundFlare
+    "BohuTANG",
+    "damozhaeva",   # DOCSUP
+    "den-crane",
+    "gyuton",       # DOCSUP
+    "gyuton",       # technical writer, Yandex
```
Comment (blinkov) [inline tests/ci/approve_lambda/app.py:59]: duplicate

File: tests/ci/approve_lambda/app.py [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+    "hagen1778",    # Roman Khavronenko, seasoned contributor
+    "hczhcz",
+    "hexiaoting",   # Seasoned contributor
+    "ildus",        # adjust, ex-pgpro
+    "javisantana",  # a Spanish ClickHouse enthusiast, ex-Carto
+    "ka1bi4",       # DOCSUP
+    "kirillikoff",  # DOCSUP
+    "kitaisreal",   # Seasoned contributor
```
Comment (blinkov) [inline tests/ci/approve_lambda/app.py:67]: employee

File: tests/ci/approve_lambda/app.py [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+    "nikvas0",
+    "nvartolomei",
+    "olgarev",      # DOCSUP
+    "otrazhenia",   # Yandex docs contractor
+    "pdv-ru",       # DOCSUP
+    "podshumok",    # cmake expert from QRator Labs
+    "s-mx",         # Maxim Sabyanin, former employee, present contributor
+    "sevirov",      # technical writer, Yandex
+    "spongedu",     # Seasoned contributor
+    "ucasFL",       # Amos Bird's friend
```
[the remark this answers is not in the document]
Comment (vdimir) [reply tests/ci/approve_lambda/app.py:82]: Maybe check here can be case-insensitive? GitHub redirects usernames that differs only with case to default one, so I suppose it's impossible to have different users for them.

[the remark this answers is not in the document]
Comment (ucasfl) [reply tests/ci/approve_lambda/app.py:82]: Btw, what is the purpose of the PR :)

[the remark this answers is not in the document]
Comment (ucasfl) [reply tests/ci/approve_lambda/app.py:82]: > Maybe check here can be case-insensitive? GitHub redirects usernames that differs only with case to default one, so I suppose it's impossible to have different users for them.

Yes, it's case-insensitive.

[the remark this answers is not in the document]
Comment (alesapin) [reply tests/ci/approve_lambda/app.py]: `> Btw, what is the purpose of the PR :)`
Automatically approve github actions run.

**Pull request merged**